src/new-client/src/app/site/[site]/details/edit-information-for-citizens/add-information-for-citizens-form.tsx

```diff
@@ -48,16 +52,57 @@ const AddInformationForCitizensForm = ({
     replace(`/site/${site}/details`);
   };
 
+  const isValidTextInput = (text: string): boolean => {
+    const urlRegex = new RegExp(
+      '([a-zA-Z0-9]+://)?([a-zA-Z0-9_]+:[a-zA-Z0-9_]+@)?([a-zA-Z0-9.-]+\\.[A-Za-z]{2,4})(:[0-9]+)?([^ ])+',
+    );
+    if (urlRegex.test(text)) {
+      return false;
+    }
+
+    const specialCharacterRegex = /^[-\w \.\,\-]+$/;
+    return specialCharacterRegex.test(text);
+  };
+
+  const handleTextInputUpdate = (inputLength: number): void => {
+    setTextInputLength(inputLength);
+  };
+
   return (
     <form onSubmit={handleSubmit(submitForm)}>
-      <FormGroup legend="Information for citizens">
-        <div className="nhsuk-form-group">
-          <TextArea
-            label="What information would you like to include?"
-            {...register('informationForCitizen')}
-          />
-        </div>
-      </FormGroup>
+      <div
+        className="nhsuk-character-count"
+        data-module="nhsuk-character-count"
+        data-maxlength="150"
+      >
+        <FormGroup
+          legend="Information for citizens"
+          error={errors.informationForCitizen?.message}
+        >
+          <div className="nhsuk-form-group">
+            <TextArea
+              label="What information would you like to include?"
+              maxLength={maxLength}
+              {...register('informationForCitizen', {
+                validate: value => {
+                  if (!isValidTextInput(value)) {
+                    return "Text cannot contain a URL or special characters outside of '.' ',' '-'";
+                  }
+                },
+                onChange: e => {
```

Comment:
If we use the RHF methods I've mentioned elsewhere, we shouldn't need this onChange method at all :)